react/Viewer/Readme.md

```diff
@@ -1,8 +1,12 @@
+Note :
+
+- The `Viewer` can be used only in a `React Application`. You can't use it with `Preact`.
+- If you want to use the `Viewer` in your app and if you use Webpack, you have to add an alias in your configuration :`react-pdf : react-pdf/dist/entry.webpack.js`
```

Comment:
```suggestion
- If you want to use the `Viewer` in your app and if you use Webpack, you should add the following alias in your configuration : `react-pdf : react-pdf/dist/entry.webpack.js`. This will make sure a web-worker is used to render PDF files.
```